README.md

```diff
@@ -1,44 +1,42 @@
-[![Stories Ready](https://badge.waffle.io/quran/quran.com-frontend.svg?label=ready&title=Ready)](http://waffle.io/quran/quran.com-frontend)
-[![Stories In Progress](https://badge.waffle.io/quran/quran.com-frontend.svg?label=in%20progress&title=In%20Progress)](http://waffle.io/quran/quran.com-frontend)
-[![Stories In Review](https://badge.waffle.io/quran/quran.com-frontend.svg?label=in%20review&title=In%20Review)](http://waffle.io/quran/quran.com-frontend)
-
-
-## Quran.com
-This is the project soon to be the Quran.com facing site. This is built in
-Reactjs + Redux + Expressjs + Webpack. It is isomorphic (javascript shared
-between both the server and the client) for SEO reasons.
+# quran.com
+This is the project soon to be the quran.com facing site. This is built in
+Reactjs + Redux + Expressjs + Webpack. It is isomorphic which is fancy speak
+to say that the javascript is shared between both the server and the client.
 
 [![Dependency Status](https://david-dm.org/quran/quran.com-frontend.svg)](https://david-dm.org/quran/quran.com-frontend)
 [![Code Climate](https://codeclimate.com/github/quran/quran.com-frontend.png)](https://codeclimate.com/github/quran/quran.com-frontend)
 
-#### Getting started
-Simply clone this repo, then run `npm install` to install all the required node_modules.
-From there, you are ready to go!
 
-#### Developing
-To start the app, run `npm run dev` which will
-run both the server and the client (webpack) to compile upon edits. Go to localhost:8001 in your browser, not 8000 (that is just the express server).
+### Getting started
+If you hate reading then skip to the 'Lazy synopsis' sub-section. If you've got 5 minutes to spare, read this entire section.
```

Comment:
Why not just call it `TL;DR` and have it here and let the reader continue if they need to?